src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetExtensionTests.ps1

```diff
@@ -15,297 +15,140 @@
-        $keyVaultResourceId = $keyVault.ResourceId;
-        $keyEncryptionKeyUrl = $kek.Key.kid;
-
-        # SRP
-        $stoname = 'sto' + $rgname;
-        $stotype = 'Standard_GRS';
-        New-AzureRMStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype;
-        $stoaccount = Get-AzureRMStorageAccount -ResourceGroupName $rgname -Name $stoname;
-
-        # NRP
-        $subnet = New-AzureRMVirtualNetworkSubnetConfig -Name ('subnet' + $rgname) -AddressPrefix "10.0.0.0/24";
-        $vnet = New-AzureRMVirtualNetwork -Force -Name ('vnet' + $rgname) -ResourceGroupName $rgname -Location $loc -AddressPrefix "10.0.0.0/16" -Subnet $subnet;
-        $vnet = Get-AzureRMVirtualNetwork -Name ('vnet' + $rgname) -ResourceGroupName $rgname;
-        $subnetId = $vnet.Subnets[0].Id;
-
-        # New VMSS Parameters
-        $adminUsername = 'Foo12';
-        $adminPassword = $PLACEHOLDER;
-
-        $imgRef = Get-DefaultCRPImage -loc $loc;
-
-        $ipCfg = New-AzureRmVmssIPConfig -Name 'test' -SubnetId $subnetId;
-        $vmss = New-AzureRmVmssConfig -Location $loc -SkuCapacity 2 -SkuName 'Standard_A1' -UpgradePolicyMode 'automatic' `
-         | Add-AzureRmVmssNetworkInterfaceConfiguration -Name 'test' -Primary $true -IPConfiguration $ipCfg `
-         | Set-AzureRmVmssOSProfile -ComputerNamePrefix 'test' -AdminUsername $adminUsername -AdminPassword $adminPassword `
-         | Set-AzureRmVmssStorageProfile -OsDiskCreateOption 'FromImage' -OsDiskCaching 'None' `
-         -ImageReferenceOffer $imgRef.Offer -ImageReferenceSku $imgRef.Skus -ImageReferenceVersion $imgRef.Version `
-         -ImageReferencePublisher $imgRef.PublisherName;
-
-        $result = New-AzureRmVmss -ResourceGroupName $rgname -Name $vmssName -VirtualMachineScaleSet $vmss;
-
-        # Get
-        $vmssResult = Get-AzureRmVmss -ResourceGroupName $rgname -VMScaleSetName $vmssName;
-
-        # Get Instance View
-        $vmssInstanceViewResult = Get-AzureRmVmss -ResourceGroupName $rgname -VMScaleSetName $vmssName -InstanceView;
-
-        Set-AzureRmVmssDiskEncryptionExtension -ResourceGroupName $rgname -VMScaleSetName $vmssName `
-                                            -AadClientID $aadClientID -AadClientSecret $aadClientSecret `
-                                            -DiskEncryptionKeyVaultUrl $diskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $keyVaultResourceId `
-                                            -KeyEncryptionKeyUrl $keyEncryptionKeyUrl -KeyEncryptionKeyVaultId $keyVaultResourceId -Force
-
-        $result = Get-AzureRmVmssDiskEncryption -ResourceGroupName $rgname -VMScaleSetName $vmssName;
-        $result_string = $result | Out-String;
-
-        $result = Get-AzureRmVmssVMDiskEncryption -ResourceGroupName $rgname -VMScaleSetName $vmssName;
-        $result_string = $result | Out-String;
-
-    }
-    finally
-    {
-        # Cleanup
-        Clean-ResourceGroup $rgname
-    }
+	# Common
+	$loc = 'eastus2euap';
```

Comment:
The changes in this PR were required for the existing tests to be re-recorded, since the previous versions of these tests did not set up the prerequisites either.  In an effort to move this forward to where we both want it to be, I've added detailed instructions on how to manually set up the necessary prerequisites for these tests.   These new recordings give us parity with the prior recordings.  With the added instructions it is possible to do pre-suite setup manually as would be required for any re-recording, and sets us up to being able to fully automate pre-suite setup in our next sprint.